### DISCUSSION TITLE
Getting error using React Native example

### DISCUSSION THREAD

**Discussion by developerwill** [osamaqarem/react-native-image-colors#61] [Q&A]
I'm trying to get this working on a Expo React Native using the exact code from the example provided at: https://github.com/osamaqarem/react-native-image-colors/blob/master/example/App.js and I'm getting this error: ```
Possible Unhandled Promise Rejection (id: 0):
TypeError: null is not an object (evaluating '_module.RNImageColors.getColors')
```

**Comment by osamaqarem** [comment]
hey there, try to provide the steps you've done before reaching this problem (creating the project, installing the library, running the app etc.) this will help in troubleshooting.
